persistence/src/commonMain/db_user/com/wire/kalium/persistence/Messages.sq

```diff
@@ -306,6 +306,7 @@ AS SELECT
     Message.creation_date AS date,
     Message.visibility AS visibility,
     Message.sender_user_id AS senderUserId,
+    (Message.expire_after_millis IS NOT NULL) AS isEphemeral,
```

Comment:
You need to write migration sqm file for that change

Reply:
thanks for the reminder